hbase-it/src/test/java/org/apache/hadoop/hbase/test/IntegrationTestReplication.java

```diff
@@ -121,10 +123,10 @@ protected class ClusterID {
      */
     public ClusterID(Configuration base, String key) {
       configuration = new Configuration(base);
-      String[] parts = key.split(":");
-      configuration.set(HConstants.ZOOKEEPER_QUORUM, parts[0]);
-      configuration.set(HConstants.ZOOKEEPER_CLIENT_PORT, parts[1]);
-      configuration.set(HConstants.ZOOKEEPER_ZNODE_PARENT, parts[2]);
+      Iterable<String> parts = Splitter.on(':').split(key);
```

Comment:
Better to split to list and then use get 0, 1, 2? Or create a iterator, call next 3 times. Iterables.get will always start from the  first one and iterate...